packit/cli/propose_downstream.py

```diff
@@ -140,6 +157,25 @@ def propose_downstream(
     )
 
     for branch in branches_to_update:
+        description = None
+        rebase_bug_no = None
+
+        # BUG(Gris Ge): buggy code on accessing jumble package_config data
+        for job in api.package_config.jobs:
```

Comment:
same as in my previous comment applies here

Reply:
Please check whether V2 patch is better.